### DISCUSSION TITLE
./run lint returns nothing (OK) locally, but CI fails

### DISCUSSION THREAD

**Discussion by SHxKM** [nickjj/docker-django-example#43] [Q&A] [answered]
Title says it all. CI is complaining about “improperly ordered imports” but I don’t get those warnings when running `./run lint` locally.

**Comment by nickjj** [comment, marked as answer]
Hmm, try running `./run quality` which runs a combo of lint (flake8), format (black) and format:imports (isort).

It sounds like you'd want to run `./run format:imports` to detect it locally. I just happen to run quality in my day to day because it does all 3 in 1 shot.

**Reply by SHxKM** [reply]
Worked, thanks!
